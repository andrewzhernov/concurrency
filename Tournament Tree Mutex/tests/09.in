4 25000 10
