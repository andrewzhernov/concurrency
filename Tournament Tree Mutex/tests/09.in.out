Testing with parameters: n_threads=4, n_iters=25000, n_shareds=10
Starting 4 threads ... 
Done
Joining 4 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=4, n_iters=25000, n_shareds=10
Starting 4 threads ... 
Done
Joining 4 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=4, n_iters=25000, n_shareds=10
Starting 4 threads ... 
Done
Joining 4 threads ... 
Done
Checking test invariant ... 
Done
OK
